homeassistant/components/motion_blinds/cover.py

```diff
@@ -146,6 +152,21 @@ def is_closed(self):
         """Return if the cover is closed or not."""
         return self._blind.position == 100
 
+    @callback
+    def _push_callback(self):
```

Comment:
We don't need this method at all. Register `self.schedule_update_ha_state` as callback directly, if it will be called from sync context.

Reply:
How would I then handle the `force_refresh=False` attribute?
Because I can only register callback functions withouth arguments to the upstream MotionBlinds library right now.

Reply:
That's the default and not needed parameter.